recipe/build.sh

```diff
@@ -19,6 +19,8 @@ cmake \
   -DPython_EXECUTABLE:FILEPATH=FALSE \
   -DPython3_EXECUTABLE:FILEPATH=FALSE \
   -DWITH_PYTHON_BINDINGS:BOOL=FALSE \
+  -DWITH_PYTHON_BINDINGS_V2:BOOL=FALSE \
+  -DWITH_PYTHON_BINDINGS_V3:BOOL=FALSE \
```

Comment:
This change also gets the job done. However, I want to point out the very soon we want to be dropping Boost and building a Python version independent htcondor2 bindings. This will happen in the next few months.

Reply:
Thanks for the heads up.